spec/cases/each_with_threads_and_exceptions.rb

```diff
@@ -0,0 +1,12 @@
+require './spec/cases/helper'
```

Comment:
Cool, considered DRYing these up, but I wasn't sure if adding some abstraction would be distracting. Would you be in favor of something like this?

`spec/cases/with_exceptions.rb`
```ruby
require './spec/cases/helper'

method = ENV['METHOD']
in_worker_type = "in_#{ENV['WORKER_TYPE']}"

begin
  Parallel.public_send(method, 1..100, in_worker_type => 4) do |x|
    sleep 0.1 # so all processes get started
    print x
    raise 'foo' if x == 1
    sleep 0.1 # so now no work gets queued before exception is raised
  end
rescue
  print ' raised'
end
```